CHANGELOG.md

```diff
@@ -5,6 +5,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.37.0](https://github.com/buildkite/agent/compare/v3.36.1...v3.37.0) (2022-07-06)
+
+### Added
+
+* Agent metadata includes `arch` (e.g. `arch=amd64`) alongside `hostname` and `os` [#1691](https://github.com/buildkite/agent/pull/1691) ([moskyb](https://github.com/moskyb))
+* Allow forcing clean checkout of plugins [#1636](https://github.com/buildkite/agent/pull/1636) ([toothbrush](https://github.com/toothbrush))
+
+### Fixed
+
+* Don't blow up when a hook exports an array [#1692](https://github.com/buildkite/agent/pull/1692) ([moskyb](https://github.com/moskyb))
+* Unescape backticks when parsing env from export -p output [#1687](https://github.com/buildkite/agent/pull/1687) ([moskyb](https://github.com/moskyb))
+* Log “Using flock-file-locks experiment 🧪” when enabled [#1688](https://github.com/buildkite/agent/pull/1688) ([lox](https://github.com/lox))
+* flock-file-locks experiment: errors logging [#1689](https://github.com/buildkite/agent/pull/1689) ([KevinGreen](https://github.com/KevinGreen))
+* Remove potentially-corrupted mirror dir if clone fails [#1671](https://github.com/buildkite/agent/pull/1671) ([lox](https://github.com/lox))
+* Improve log-level flag usage description [#1676](https://github.com/buildkite/agent/pull/1676) ([pzeballos](https://github.com/pzeballos))
+
+### Changed
+
+* datadog-go major version upgrade to v5.1.1 [#1666](https://github.com/buildkite/agent/pull/1666) ([moskyb](https://github.com/moskyb))
+* Revert to delegating directory creation permissions to system umask [#1667](https://github.com/buildkite/agent/pull/1667) ([moskyb](https://github.com/moskyb))
+* Replace retry code with [roko](https://github.com/buildkite/roko) [#1675](https://github.com/buildkite/agent/pull/1675) ([moskyb](https://github.com/moskyb))
+* Refactor env.Environment from wrapping a map to being a map [#1655](https://github.com/buildkite/agent/pull/1655) ([moskyb](https://github.com/moskyb))
```

Comment:
We can leave this entry out i reckon - it's a refactor that shouldn't affect anyone other than us

Reply:
Yeah I was on the fence — if somebody starts having strange environment issues in this release, it could be a good hint. But that seems unlikley enough to not include it.

Reply:
(removed)